ledger/participant-integration-api/src/main/scala/platform/store/interfaces/TransactionLogUpdate.scala

```diff
@@ -33,9 +33,7 @@ object TransactionLogUpdate {
       effectiveAt: Timestamp,
       offset: Offset,
       events: Vector[Event],
-  ) extends TransactionLogUpdate {
-    require(events.nonEmpty, "Transaction must have at least an event")
```

Comment:
Requirement pertaining to the old IMFO stream assembly mechanism that does not hold anymore.